### ISSUE TITLE
the err msg is gone

### ISSUE THREAD

**Issue by wuqinqiang** [tech-greedy/singularity#110]
**Describe the bug**
I noticed that when an error occurred in calling go-generate-car， the err msg is gone.
such as 

![截屏2022-07-25 11 47 28](https://user-images.githubusercontent.com/36129334/180695174-4057df5c-dffd-424b-9e7c-2a4bf3f644b8.png)

if I do not  set GOLOG_FILE env，no err mesage.

**Version**
1.1.1

**To Reproduce**

**Expected behavior**

**Comment by liuziba** [comment]
Were you trying to pause or remove the related generation request?

**Comment by wuqinqiang** [comment]
> Were you trying to pause or remove the related generation request?

No.

**Comment by liuziba** [comment]
What's the error message if you set go_logfile?

**Comment by wuqinqiang** [comment]
no more err now. 

I mean， since an error occurred, there should be an error msg, even if it is not set something.

**Comment by liuziba** [comment]
I'll try to reproduce it. The only change I'm aware of that can cause it, which is added in version 1.1.0 is that, when you pause or remove a dataset, all ongoing child process will now be killed and returns an empty stderr.

**Comment by liuziba** [comment]
Confirmed that the error message is not missing for known error cases.
Adding the captured Signals in the next release so we know if it is being killed externally or by pausing or deleting.
```
2022-07-26T17:55:32.909Z [deal_preparation_service] info: Received request to update dataset preparation request. - {"dataset":"test2","action":"pause"}
2022-07-26T17:55:32.914Z [deal_preparation_worker] error: fbccd068-e7b3-4f22-a363-b7cb51f460ad - Encountered an error. - {"stderr":"","statusCode":null,"signalCode":"SIGTERM"}
```

**Comment by liuziba** [comment]
Solved in #109 , please reopen if it still happens.

**Comment by wuqinqiang** [comment]
thanks~~
